statemachine/factory.py

```diff
@@ -27,10 +32,11 @@ def __init__(cls, name: str, bases: Tuple[type], attrs: Dict[str, Any]):
         """Map of ``state.value`` to the corresponding :ref:`state`."""
 
         cls._abstract = True
+        cls._strict_states = strict_states
         cls._events: Dict[str, Event] = {}
 
-        cls.add_inherited(bases)
-        cls.add_from_attributes(attrs)
+        StateMachineMetaclass.add_inherited(cls, bases)
```

Comment:
**thought:** I didn't get why this change was required. Can you share the linter issue you've got to require this commit 540acce4d77c8b88378104be1e524302ca894fed? Reverting this will lower the diff.

Reply:
This avoids a`pylint` error: `No value for argument 'bases' in unbound method call` Pylint([E1120:no-value-for-parameter](https://pylint.readthedocs.io/en/latest/user_guide/messages/error/no-value-for-parameter.html))

I can remove it if you wish, but it's a fairly innocuous change that's in it's own commit.

Reply:
I believe this might be a false positive, potentially linked to [this issue](https://github.com/pylint-dev/pylint/issues/3268). 

I prefer maintaining the `cls` reference instead of repeatedly using the class name. Could you please revert it?

Reply:
this commit has been reverted.